courses/views.py

```
@@ -0,0 +1,38 @@
+    if created:
+        messages.success(request, f'Successfully enrolled in {course.title}')
+    else:
+        messages.info(request, f'You are already enrolled in {course.title}')
+    return redirect('courses:course_detail', slug=slug)
+
+def session_list(request, slug):
+    course = get_object_or_404(Course, slug=slug, is_active=True)
+    sessions = course.sessions.all()
+    return render(request, 'courses/session_list.html', {
+        'course': course,
+        'sessions': sessions
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)